core/platform/auth/firebase_auth_services_test.py

```
@@ -36,7 +36,7 @@
 import webapp2
 from firebase_admin import auth as firebase_auth
 from firebase_admin import exceptions as firebase_exceptions
-from typing import ContextManager, Dict, List, Optional, Tuple, Union, cast
+from typing import Any, ContextManager, Dict, List, Optional, Tuple, Union, cast
```

Ruff (0.15.0)

[warning] `typing.ContextManager` is deprecated, use `contextlib.AbstractContextManager` instead

(UP035)